starter/src/main/java/com/starter/fullstack/rest/InventoryController.java

```diff
@@ -33,5 +35,14 @@ public InventoryController(InventoryDAO inventoryDAO) {
   public List<Inventory> findInventories() {
     return this.inventoryDAO.findAll();
   }
+  /**
+   * Creates Inventory
+   * @param inventory inventory
+   * @return created inventory
+   */
+  @PostMapping("/inventory")
```

Comment:
When you fix the missing `@RequestMapping` above, you'll want to remove the path from this mapping. Don't forget to remove the `()`'s as well.